lib/service.js

```diff
@@ -160,21 +160,35 @@ class Service {
   serviceStatus(cb) {
     return this._execServiceStatus((err, response) => {
       if (err != null) return cb(err);
+
       let status = response.body.status;
+      let version = response.body.version;
+
+      if (version != null) {
+        version = chalk.whiteBright(`v${version}`);
+      }
+
       if (status === STATUS_CONSTANTS.ONLINE) {
-        status = chalk.green(STATUS_CONSTANTS.ONLINE);
+        status = chalk.greenBright(STATUS_CONSTANTS.ONLINE);
       }
       if (status === STATUS_CONSTANTS.UPDATING) {
-        status = chalk.yellow(STATUS_CONSTANTS.UPDATING);
+        status = chalk.yellowBright(STATUS_CONSTANTS.UPDATING);
       }
       if (status === STATUS_CONSTANTS.NEW) {
-        status = chalk.cyan(STATUS_CONSTANTS.NEW);
+        status = chalk.cyanBright(STATUS_CONSTANTS.NEW);
       }
       if (status === STATUS_CONSTANTS.ERROR) {
-        status = chalk.red(STATUS_CONSTANTS.ERROR);
+        status = chalk.redBright(STATUS_CONSTANTS.ERROR);
       }
-      logger.info('Service status: %s', status);
-      return cb(null, response.body.status);
+
+      // Print version if attached to response
+      if (response.body.version != null) {
+        logger.info('Service status: %s [%s]', status, version);
+      } else {
+        logger.info('Service status: %s', status);
+      }
+
+      return cb(null, response.body.status, response.body.version);
```

Comment:
Wouldn't it be better if the callback receives only two arguments - error as first and data as second? It seems more like the node.js way. And data would be an object containing status, version and whatever else may be added in the future.